cfme/tests/v2v/test_v2v_migrations.py

```diff
@@ -479,3 +481,55 @@ def _is_migration_started():
     for vm in vms:
         soft_assert(request_details_list.is_successful(vm) and
          not request_details_list.is_errored(vm))
+
+
+def test_migration_long_name(request, appliance, v2v_providers, host_creds, conversion_tags):
+    """Test to check VM name with 64 character should work"""
+    source_datastores_list = v2v_providers[0].data.get('datastores', [])
```

Comment:
Please rebase and checkout the modifications to the v2v fixtures - they're returning namedTuple now, so here you can reference `v2v_providers.vmware_provider` or v2v_providers.rhvem_provider`, and it will be much more explicit which provider you ware working with.

Same applies to the fixtures that provide form data and a list of VM's. They have a unique NameTuple class now as well.

Reply:
rebased with changes, it looks clean now. thanks! :+1: